README.rst

```diff
@@ -101,8 +148,8 @@ resources and everyone in between!
 How do I access the data?
 -------------------------
 
-There are several ways to access PUDL outputs. For more details you'll want
-to check out `the complete documentation
+There are several ways to access the information in the PUDL Data Warehouse.
+For more details you'll want to check out `the complete documentation
```

Comment:
feels weird to link to the documentation which literally just brings you back to this page.

Reply:
But maybe it's a weird readme thing?

Reply:
Oh yeah that's bad. That should probably link to the data access page?